### ISSUE TITLE
[Feature Request] Individual volume sliders for the Intercom channels

### ISSUE THREAD

**Issue by Pi123263** [michail-nikolaev/task-force-arma-3-radio#1497]
From the TFAR Discord:
> The ask here was for there to be two different volume sliders. One for Crew, one for cargo/ whatever. While yes, as you say, you can't listen to two different intercoms, it was so that you dont need to change the level depending on what you are doing. 
So using the rhs Abrams as an example, the turbine is pretty loud. So have a crew com be loud enough to cover that would be useful. But then your tank is blown up, and you end up in the back of a Bradley. Now in the cargo, you dont need the intercom to be as loud. 
The feature request/enhancement is that there is the ability to have different volume levels depending on the intercom. This is instead of having to go into options to change it each time

(https://discordapp.com/channels/233658633957802016/233658633957802016/677135681876918306)

_TL;DR: Individual volume slider for the individual Intercom channels to reduce the amount of mid-game setting changing._

**Comment by severgun** [comment]
Seems like there is no mandatory intercom config. So every vehicle is special and no way to sort which is "crew" channel and which is "cargo". This is just localized names of actions.

**Comment by Garfield0003** [comment]
Are the actions consistent. So for vic config, is there an option called "crew" that is universal? Or is it up to the person implementing it to name the intercom?

Hopefully that made sense to, happy to elaborate if required

**Comment by severgun** [comment]
There is no strict rules. Addon maker can define channels whatever he wants.

https://github.com/michail-nikolaev/task-force-arma-3-radio/blob/1ea59fe2df73010603c19737a517ac1c5a2cb635/addons/core/intercomConfig/config.cpp#L34-L38

Here `displayName` is just localized name.
https://github.com/michail-nikolaev/task-force-arma-3-radio/blob/53edae8e1716dd56b37bd0a116abb4cfce4e0210/addons/core/stringtable.xml#L2819-L2820

So in my hypothetical addon of taxi cab I can name channel with ID `TFAR_IntercomSlot_0` as `My Awesome DanceFM` and stream music in my fancy taxi cab.
